src/translations/common/titles.ts

```diff
@@ -10,6 +10,10 @@ export const Titles = orgSpecificTranslations(TitlesInternal, {
     siteTitle: _('Reis Reisesøk', 'Reis Travel Search', 'Reis Reisesøk'),
   },
   fram: {
-    siteTitle: _('FRAM Reisesøk', 'FRAM Travel Search', 'FRAM Reisesøk'),
```

Comment:
Are we preferring Reisesøk or Reiseplanlegger here?

Reply:
> Når vi skal bytte fra FRAM Reisesøk og FRAM Nettbutikk til bare FRAM-logoen i navbaren på nettsidene (trenger ikke gjøre logoen større enn plassen som blir brukt nå), kan vi fjerne at det står FRAM Reiseplanlegger og FRAM Nettbutikk i footeren også.
I i footeren kan det stå bare Reiseplanlegger og Nettbutikk i første spalten (hvor det står FRAM Reisesøk og FRAM Nettbutikk nå), og i andre spalten redigere Hjelp til reiseplanleggeren til Kontakt oss (skrive Kontakt oss i nettbutikken også)

I think PO would prefer Reiseplanlegger ☝️

Reply:
Fine as it is now then 👌 You can do the final review if you have time 🤝